app/geant-exporter/GeantPhysicsTableWriter.hh

```diff
@@ -0,0 +1,72 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file GeantPhysicsTableWriter.hh
+//! \brief Write Geant physics tables into a ROOT file
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <iostream>
+#include <vector>
+#include <memory>
+
+#include "io/GeantPhysicsTable.hh"
+
+// ROOT
+class TFile;
+class TTree;
+
+// Geant
+class G4VProcess;
+class G4VEmProcess;
+class G4VEnergyLossProcess;
+class G4VMultipleScattering;
+class G4ParticleDefinition;
+class G4PhysicsTable;
+
+using namespace celeritas;
```

Comment:
I think the geant_exporter namespace is fine as it is. And the `using namespace celeritas` was a placeholder (since I didn't know all objects I would need while creating the table writer). I guess now I can simply input the used objects manually (ie `using celeritas::class`), as it is done in the tests. Does this sounds reasonable to you?

Reply:
Any `using` stuff needs to be either inside a class, or inside a `.cc` (or `.cu`) translation unit. The idea is that the existence of a name inside a particular namespace should be obvious wherever you use it.

Reply:
Done.